dash/dash.py

```diff
@@ -931,6 +1019,11 @@ def _validate_layout(self):
 
         component_ids = {layout_id} if layout_id else set()
         for component in to_validate.traverse():
+            if (
```

Comment:
Normally there is no `()` after if in python.

Reply:
Not sure how else to style since the if statement is > 80 characters. PEP isn't super specific about what to do here: https://www.python.org/dev/peps/pep-0008/#multiline-if-statements.